examples/export-pdf.js

```diff
@@ -64,6 +65,7 @@ exportButton.addEventListener('click', function() {
       // Reset original map size
       map.setSize(size);
       map.getView().fit(extent, {size});
```

Comment:
Wouldn't it be enough to just change this line to
```js
map.getView().fit(extent, {
  size: size,
  constrainResolution: false
});
```
Then you could remove the rest of the changes.

Reply:
Totally right. Just provided your suggestion in the commit. Thanks.